install.sh

```diff
@@ -177,6 +176,24 @@ legacy_transformations_end() {
     git config --global --unset githooks.autoupdate.updateCloneBranch
     git config --global --unset githooks.previous.searchdir
 
+    # Remove legacy registration file (we moved it to another location)
+    if [ -f "$INSTALL_DIR/autoupdate/registered" ]; then
+        rm -rf "$INSTALL_DIR/autoupdate"
+    fi
+    return 0
+}
+
+############################################################
+# Function to dispatch to all legacy transformations
+#   at the end of a install into a repository
+#
+# Returns:
+#   1 when failed, 0 otherwise
+############################################################
+legacy_transformations_repo_end() {
+
+    git -C "$1" config --local --unset githooks.autoupdate.registered
```

Comment:
Do we re-register already register repos on the first install/update here?
If not, should this need migration from the old config to the new config?

Reply:
We copied the registered list which already contains all reigstered repos so far, and any single install repo will still have old hooks (if it got not hit by the `install into existing`)  meaning it wont get registered... 

Ups we have forgotten a legacy transformation which would ask to search for single repo installs and then making them `non-single`.... Maybe we could add that, or is it not so important?

Reply:
If the update touches them and they are still single then yeah making them non-single sounds sensible to me.